runtime/src/lib.rs

```diff
@@ -293,7 +293,9 @@ impl pallet_grandpa::Config for Runtime {
 	type MaxSetIdSessionEntries = ConstU64<0>; // used for equivocation
 }
 
-pub const EPOCH_DURATION_IN_BLOCKS: u32 = prod_or_fast!(12 * HOURS, 15);
+const BLOCKS_FOR_FASTER_EPOCH: u32 = if cfg!(feature = "zombienet") { 45 } else { 15 };
```

Comment:
The new flag looks good to me, I like having a different const for calculating "faster" variants instead of packing more stuff into the production one